osu.Server.Spectator/Services/SharedInterop.cs

```diff
@@ -93,6 +94,9 @@ private async Task<string> runCommand(HttpMethod method, string command, dynamic
             }
             catch (Exception e)
             {
+                if (e is SharedInteropRequestFailedException interopException && !interopException.AllowRetry)
```

Comment:
i probably would have just inlined the status code comparisons here rather than separate `AllowRetry` property, easier to read. seems fine otherwise.